ui/k-info-panel/main.kytos

```diff
@@ -0,0 +1,307 @@
+            
+            let table, tr, td, value
+            let index = i
+
+            table = document.getElementById("maintenance-table-list-windows")
+            // Hide the table to prepare for rendering.
+            table.style.display = "none"
+            body = table.getElementsByTagName("tbody")
+            tr = body[0].getElementsByTagName("tr")
+
+            // For every row in the table
+            for(let j = 0; j < tr.length; j++) {
+                // Get the column correspoding to the property header to filter on.
+                td = tr[j].getElementsByTagName("td")[index]
+
+                // If the column doesn't match the filter, hide the row.
+                if(td) {
+                    value = td.textContent || td.innerText
+                    if(value.toUpperCase().indexOf(filter) < 0) {
+                        tr[j].style.display = "none"
+                    }
+                }
+            }
+            // Display the table.
+            table.style.display = ""
+        }
+     },
+     /*
+         Clear the filters applied on the table.
+     */
+     clearTableFilter: function() {
+        let table = document.getElementById("maintenance-table-list-windows")
+        let body = table.getElementsByTagName("tbody")[0]
+        let tr = body.getElementsByTagName("tr")
+        // For every row in the table, display it.
+        for(let i = 0; i < tr.length; i++) {
+            tr[i].style.display = ""
+        }
+     },
+     /*
+         Gets the maintenance windows data by using the /kytos/maintenance endpoint.
+     */
+     listMaintenanceWindows: function() {
+        var _this = this
+        var data_rows = []
+        this.render_value = 0
+
+        var request = $.ajax({
+            url: this.$kytos_server_api + "kytos/maintenance",
+            type: "GET",
+            dataType: "json",
+            contentType: "application/json"
+        })
+        request.done(function(data) {
+          $.each(data, function(i, window) {
+              if(window) {
+                  // Convert numberic status (0,1,2) to their correspoding words. 
+                  // If an status is not 0,1,2 then it will display it as a number (default).
+                  let status = window.status
+                  if(window.status == '0') {
+                      status = 'Pending'
+                  } else if(window.status == '1') {
+                      status = 'Running'
+                  } else if(window.status == '2') {
+                      status = 'Finished'
+                  }
+
+                  let row = {
+                      "id": window.id,
+                      "start": window.start,
+                      "end": window.end,
+                      "description": window.description,
+                      "items": window.items.join(", "),
+                      "status": status
+                  }
+                  data_rows.push(row)
+              }
+          })
```

Comment:
Could you also the case where the request might fail with `request.fail`, and emit an `alert` withe more information in case that ever happens?